.gitignore

```diff
@@ -35,4 +35,4 @@ bin/
 build/
 /borges
 /borges.exe
-
+go.sum
```

Comment:
why put the go.sum in gitignore? borges is an app

Reply:
- go.sum is generate by go build, do we need it? why?
- with go modules do we still distinguish between apps and packages?

Reply:
If we provide a `go.mod`, we should provide a `go.sum`, specially for an app where reproducible builds matter. I just don't think we should provide a `go.mod` as long as we have dep for dependency management, since this is not a dependency people is going to import.